### PR TITLE
Feature: Spike Contrast

### PR DESCRIPTION
**Pull request by dizcza**
This pull request should not be accepted - it's made to see which files are affected in the original pull request https://github.com/NeuralEnsemble/elephant/pull/338

### CODE REVIEW

File: elephant/spike_train_synchrony.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    Raises
+    ------
+    ValueError
+        If `bin_shrink_factor` is not in (0, 1) range.
+
+        If the input spike trains constist of a single spiketrain.
+
+        If all input spike trains contain no more than 1 spike.
+    TypeError
+        If the input spike trains is not a list of neo.SpikeTrain objects.
```
Comment (kohlerca) [inline elephant/spike_train_synchrony.py]: missing backticks in `neo.SpikeTrain`

File: elephant/spike_train_synchrony.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Functions to measure the synchrony of several spike trains
+(:cite:`synchrony-ciba2018spike`).
```
Comment (dizcza) [inline elephant/spike_train_synchrony.py]: move the citation to the function

File: doc/bib/elephant.bib [hunk 3]
```diff
@@ -115,6 +115,16 @@ @article{Torre16_e1004939
   doi={10.1371/journal.pcbi.1004939}
 }
 
+@article{ciba2018spike,
```
Comment (mdenker) [inline doc/bib/elephant.bib]: Ciba18_136

File: elephant/spike_train_synchrony.py [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+Synchrony Measures
+------------------
+
+.. autosummary::
+    :toctree: toctree/spike_train_synchrony/
+
+    spike_contrast
+
+
+:copyright: Copyright 2015-2016 by the Elephant team, see `doc/authors.rst`.
```
Comment (mdenker) [inline elephant/spike_train_synchrony.py]: 2015-2020

File: doc/bib/elephant.bib [hunk 5]
```diff
@@ -115,6 +115,16 @@ @article{Torre16_e1004939
...
+  title={Spike-contrast: A novel time scale independent and multivariate measure of spike train synchrony},
+  author={Ciba, Manuel and Isomura, Takuya and Jimbo, Yasuhiko and Bahmer, Andreas and Thielemann, Christiane},
```
Comment (mdenker) [inline doc/bib/elephant.bib]:
```suggestion
  author={Ciba, M. and Isomura, T. and Jimbo, Y. and Bahmer, A. and Thielemann, C.},
```

File: doc/bib/elephant.bib [hunk 6]
```diff
@@ -115,6 +115,16 @@ @article{Torre16_e1004939
...
+  year={2018},
+  journal={Journal of Neuroscience Methods},
```
Comment (mdenker) [inline doc/bib/elephant.bib]: J. Neurosci. Meth.

**Pull request closed without merging**